torchao/quantization/quant_primitives.py

```diff
@@ -41,12 +41,18 @@ class MappingType(Enum):
     we'll use (-10.2, 10.2) as the range for floating point and map that to (-8, 7)
     e.g. scale = (10.2 - (-10.2)) / (7 - (-8))
 
+    SYMMETRIC_MAX_POS_NEG is a variant of symmetric mapping, where the scale is the max of smin
```

Comment:
how about `SYMMETRIC_NO_CLIPPING_ERR`? also cc @TiRune if there is suggestion for the name